src/modules/tokens/models/tokens.filter.args.ts

```diff
@@ -9,3 +20,22 @@ export class TokensFiltersArgs {
     @Field({ defaultValue: false })
     enabledSwaps: boolean;
 }
+
+@InputType()
+export class TokensFilter {
+    @Field(() => [String], { nullable: true })
+    identifiers: string[];
+    @Field({ nullable: true })
+    type: string;
```

Comment:
if you put the field as `nullable`, you have to add the `?` as well so that the Typescript can know it